tools/qualimap.wdl

```diff
@@ -102,6 +116,6 @@ task rnaseq {
     parameter_meta {
         bam: "Input BAM format file to generate coverage for"
         gencode_gtf: "A GTF format features file containing Gencode features"
-        strand: "Strand information for RNA-seq experiments. Options: [strand-specific-forward, strand-specific-reverse, non-strand-specific]"
+        provided_strand: "Strand information for RNA-seq experiments. Options: [strand-specific-forward, strand-specific-reverse, non-strand-specific]"
```

Comment:
Does this need to be updated to the new capitalized values?

Reply:
Yes it does. Looks like that parameter info was already out of date before the change, which is why I missed it. Good catch.

Reply:
Which actually, I've noticed a lot of meta info was already out of date. I'll need to include something for that in WDLdoc. I'll go through all the meta info and fix anything that's out of date before merging. Do we care about requiring everything to have meta data? Or only be concerned with things that are wrong?

Reply:
I think it should be considered a warning. Maybe wdldoc can have a `--strict` option.